packages/components/src/models/UnoCard.ts

```diff
@@ -0,0 +1,32 @@
+import type { Card } from '@models/Card';
+
+export const ALL_UNO_COLORS = ['red', 'blue', 'green', 'yellow'] as const;
+export type UnoCardColor = typeof ALL_UNO_COLORS[number];
+
+export const ALL_UNO_VALUES = [
```

Comment:
I find this much more elegant than using enums.
Let me know what you think, and if you are ok, I can refactor PlayingCard as well